Updating some K8s sunset language (#6478)
PostHog/posthog.com: contents/docs/self-host/index.mdx

import Disclaimer from './\_snippets/disclaimer.mdx'
import Sunset from '../self-host/\_snippets/sunset-disclaimer.mdx'

> > **This page covers our open-source Docker compose deployment**, which is available under a MIT license, [without guarantee](/docs/self-host/open-source/disclaimer). We have [sunset our paid Kubernetes deployment](/blog/sunsetting-helm-support-posthog) and no longer offer support for Kubernetes deployments.

Reviewer: ```suggestion
> **This page covers our open-source Docker compose deployment**, which is available under a MIT license, [without guarantee](/docs/self-host/open-source/disclaimer). We [sunset our paid Kubernetes deployment](/blog/sunsetting-helm-support-posthog) and no longer offer support for them.
```
